Source/WebKit/Shared/Cocoa/CoreIPCDDScannerResult.serialization.in

```diff
@@ -22,7 +22,7 @@
 
 #if ENABLE(DATA_DETECTION)
 secure_coding_header: <pal/cocoa/DataDetectorsCoreSoftLink.h>
-[WebKitSecureCodingClass=PAL::getDDScannerResultClass()] webkit_secure_coding DDScannerResult {
+[WebKitSecureCodingClass=PAL::getDDScannerResultClass(), AdditionalEncoder=StreamConnectionEncoder] webkit_secure_coding DDScannerResult {
```

Comment:
I'm a little surprised a DDScannerResult is sent via a StreamConnectionEncoder.  Is this really needed?  Same with DDActionContext, CNPhoneNumber, and CNPostalAddress.

Reply:
The way dictionary and array work, every type needs to have the same capabilities.
Bring in one, bring in all.

Perhaps we'll find time to improve that later!